### ISSUE TITLE
Error messages in xsd?

### ISSUE THREAD

**Issue by ue71603** [VDVde/OJP#111]
As I understand it, the error messages like

STOPEVENT_DATEOUTOFRANGE | There are   no timetables available for the requested date.
-- | --
STOPEVENT_LOCATIONUNKNOWN | The   location (address, stop etc.) for which stop events have been requested is   unknown.
STOPEVENT_LOCATIONUNSERVED | There   is no public transport service available at all at the locations (address,   stop etc.) for which stop events have been requested.
STOPEVENT_NOEVENTFOUND | No departure/arrival   could be found within the requested period of time that meets the given   restrictions.

Are only mentioned in the standard. Should we put them into the XSD as well?

**Comment by sgrossberndt** [comment]
Probably not as an enumeration as this won't be all possible codes, but we could add them as a comment/annotation.

**Comment by ue71603** [comment]
Let's try as an enum. We can update enums fast by now.

**Comment by ue71603** [comment]
Addressed in: https://github.com/VDVde/OJP/pull/203

**Comment by sgrossberndt** [comment]
We have to discuss this change again as I wrote in [this comment](https://github.com/VDVde/OJP/pull/203#issuecomment-1227801566)

1. This list currently only consists of `StopEventResponse` from the documentation, which now is `OJPStopEventDelivery`. But apparently when I removed unused structures in [0168dcd4](https://github.com/VDVde/OJP/commit/0168dcd4#diff-93bff6186bd1e221585808083d9cbb5da7cbddfab534ebf013679ff3d5596274L152) it was overlooked these `ErrorMessage`s are not part of the output at all.
I wonder now whether this was the correct choice back then to simply remove these structures - how are error messages regarding a whole response transported now? The current state seems to only be able to transport an error message for a single `StopEventResult`, but not for the whole `OJPStopEventDelivery`. Do we have to re-add `ErrorMessage` at a base delivery level?

2. Is `Code` in `ErrorMessageStructure` the correct place to document all the codes for all the responses or should they rather be documented each in the `ErrorMessage` where they may occur, e.g. https://github.com/VDVde/OJP/blob/changes_for_v1.1/OJP/OJP_StopEvents.xsd#L194

3. Currently there are only the error codes of some stopEvent documented in the xsd in this commit, the documentation has a lot more, e.g. `STOPEVENT_LASTSERVICEOFTHISLINE`, `FARES_STOPPOINTUNKNOWN`, ... which are missing here.
